go.mod

```diff
@@ -0,0 +1,139 @@
+module starknet-p2p-tests
+
+go 1.23
```

Comment:
go 1.23.0

Needed to add the .0 to run go mod tidy etc

Reply:
I guess setting the patch version isn't recommended, and from my understanding, Go uses the latest patch version automatically. Anyway, `go mod tidy` works fine for me with just `go 1.23.` 🤔